doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -67,13 +67,13 @@ The primary interface between the Physical / Virtual Infrastructure and any
 container-relevant components is the Kubernetes Node Operating System. This is
 the OS within which the container runtime exists, and within which the
 containers run (and therefore, the OS whose kernel is shared by the referenced
-containers). This is shown in Figure 3-1 below.
-
-.. image:: ../figures/ch03_hostOS.png
-   :alt: "Figure 3-1: Kubernetes Node Operating System"
+containers). This is shown in :numref:`Kubernetes Node Operating System` below.
 
+.. figure:: ../figures/ch03_hostOS.png
+   :alt: Kubernetes Node Operating System
+   :name: Kubernetes Node Operating System
 
-**Figure 3-1:** Kubernetes Node Operating System
+   Kubernetes Node Operating System
 
 The Kubernetes Node OS (as with any OS) consists of two main components:
 
```

Comment:
Not for this line but LIne 153 -- I think we have deleted "req" as a prefix from all requirements. For example, line 153 states, "To fulfil ``req.inf.vir.01`..." and this should be "inf.vir.01"

Reply:
Done in [fea5914](https://github.com/cntt-n/CNTT/pull/3065/commits/fea59141befa0e4eb69b928e1807f3f6acab1556).